### ISSUE TITLE
Undefined variables in modules

### ISSUE THREAD

**Issue by nfrerebeau** [crp2a/Al2O3_AnalysisApp#7]
Some modules use variables only defined in `app/global.R`.

**Comment by nfrerebeau** [comment]
No, it's just a note when checking the package. There's no real rush to change this, but it seems to me that it would be more neat.

**Comment by RLumSK** [comment]
Thanks for the feedback. OK, this makes sense, although I don't know a 100% why it throws that error.

**Comment by RLumSK** [comment]
@nfrerebeau I also found the reason for that errors, and they are all related to the global (`<<-`) assignment operators. Well, I guess to get rid of them, a little more work is needed. It worked nicely without a package, but within a package, this can lead to all kinds of problems. I do remember that was the reason why I never made the app part of RLumShiny. Anyway, it is doable. I was just too lazy.
